app/models.py

```diff
@@ -272,13 +172,43 @@ class Meta:
                      many=True)
 
 
-'''
-meta_schema = MetaSchema()
-companies_schema = CompaniesSchema()
-x = Companies.query.filter_by(company_id=1).first()
-companies_schema.dump(x)
+class CompaniesValidationSchema(ma.SQLAlchemySchema):
+    class Meta:
+        model = Companies
+        include_fk = True
 
-{"branches": [],"city_name": "Rotterdam","company_id": 1,"company_name": "Digital Growth Agency","company_size": "11-50","disciplines": ["Conceptontwikkeling","Conversie-optimalisatie","Strategie","Web development","Webdesign"],"logo_image_src": "https://eguide.nl/media/output/100_100/DIG_logo_trans.png","region": "Zuid-Holland","tags": [],"website": "http://www.digitalgrowthagency.nl/","year": 2019}
+    # List of regions and city_sizes for validation with CompaniesValidationSchema
+    regions = ['Remote', 'Drenthe', 'Flevoland', 'Friesland', 'Gelderland', 'Groningen', 'Limburg',
```

Comment:
Kind of like an ENUM right? 

At least we nailed down what we're willing to accept